src/register/RegistrationPage.jsx

```diff
@@ -229,6 +230,21 @@ class RegistrationPage extends React.Component {
     }
   }
 
+  handleSuggestionClick = (suggestion) => {
+    const fieldName = 'username';
+    const payload = {
+      username: suggestion,
+    };
+    this.setState({
+      ...payload,
+      fieldName,
+      updateFieldErrors: false,
+      updateAlertErrors: false,
+    }, () => {
+      this.validateInput(fieldName, suggestion, payload, false);
```

Comment:
why do we need to validate the suggestion? is it not already verified before backend sends it to the frontend?

Reply:
@zainab-amir It's just to clear out the error message and username suggestions.